packages/page-staking/src/Targets/Summary.tsx

```diff
@@ -81,7 +81,7 @@ function Summary ({ avgStaked, lastEra, lowStaked, minNominated, minNominatorBon
       <section className='media--800'>
```

Comment:
I think we should do this change in the origin when values are calculated, not directly on views:
* it'll be much easier for the reader not to miss that change, that we have some custom implementation of our targets/inflation calculation,

Hence my proposal is revert those files and instead make the change in file https://github.com/Cardinal-Cryptography/apps/blob/alephzero/packages/react-hooks/src/useInflation.ts:
* depend on condition `'yearlyInflationInTokens' in inflationParams` as this file already does, and multiply if this condition is `true` , returned `stakedReturn` by 0.9

Reply:
Done, now it is computed in better place, and inflation is not affected. Thanks!